.travis.yml

```diff
@@ -5,8 +5,11 @@ sudo: false
 install:
       - make setup
       - make update
+      - nvm install 6.10
+      - nvm use 6.10
 before_script:
-      - make lint
+      - make lint-python
+      - make lint-js
```

Comment:
While developing, we run make lint, which runs make lint-js-fix to automatically fix some JS linting errors. We don't want to make these fixes in Travis, so we just run make lint-js.